packages/react-chat/src/views/ChatWindow/index.tsx

```diff
@@ -32,16 +36,17 @@ const ChatWindow: React.FC = () => {
   );
 
   return (
-    <ChatWindowContainer>
+    <ChatWindowContainer className={className}>
       <Chat
+        autostart={!!config.autostart}
         title={assistant.title}
         description={assistant.description}
         image={assistant.image}
         avatar={assistant.avatar}
         withWatermark={assistant.watermark}
         startTime={state.session.startTime}
         hasEnded={runtime.isStatus(SessionStatus.ENDED)}
-        isLoading={!state.session.turns.length}
+        isLoading={false}
```

Comment:
I don't think we want this to always be `false`

Reply:
OH this is here  by mistake 😓